packages/wrangler/src/dev-registry.ts

```diff
@@ -205,7 +229,8 @@ export async function getBoundRegisteredWorkers({
 	const filteredWorkers = Object.fromEntries(
 		Object.entries(workerDefinitions || {}).filter(
 			([key, _value]) =>
-				serviceNames.includes(key) || durableObjectServices.includes(key)
+				key !== name && // Always exclude current worker to avoid infinite loops
```

Comment:
Mind explaining this to me? What loop is possible here? Was this a problem before this PR?